.github/workflows/get-secretmanager-secrets-it.yml

```diff
@@ -0,0 +1,47 @@
+name: get-secretmanager-secrets Integration
+on: [push]
+jobs:
+  gcloud:
+    name: with setup-gcloud
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ./setup-gcloud
```

Comment:
I cannot get this to work. I'm not sure what's going on, but SM doesn't find the ADC. Weirdly, it works fine running locally with gcloud installed, so I think it's something in the action.

Reply:
This is because the setup-gcloud action doesn't create the ADC. There a PR open for adding this which has stalled, feel free to borrow inspiration from it and add it into this PR: https://github.com/GoogleCloudPlatform/github-actions/pull/30

Reply:
Done in https://github.com/GoogleCloudPlatform/github-actions/pull/53/commits/b060325aeba4554d6e9983f16f84411c8c838e13. I took a slightly different approach with using a boolean field instead. There's no value in having a user supply a filepath because that volume needs to be shared among all builds steps for the ADC to actually work. Instead, I write it to the shared workspace. 

Default behavior is off, so it's fully BC. I also documented it, wrote tests, and updated the action specification.